uiowa/uiowa
# [Bug] Spacing fix for grids and lists

Resolves https://github.com/uiowa/uiowa/issues/7135

This pr adds the `list-container--list` class to views meant to display as lists ensuring proper spacing between each card. 

# How to test

```
ddev blt frontend && ddev blt ds --site=admissions.uiowa.edu && ddev drush @admissions.local uli /student-profiles/ainsley-hiestand
```
- Confirm sidebar grid renders without spaces
- Go to https://admissions.uiowa.ddev.site/international-student-ambassadors and confirm no visual regressions.  Try adjusting to list and confirm that space between each card is added. 
- Go to /counselors, /news, /tags/admissions-website, enable /admin/config/sitenow/sitenow-people & /counselor add two events and go to /events and confirm spacing

Code review and think of any that I may have missed in exclusion list.
